org.eclipse.xtext.ui.shared/src/org/eclipse/xtext/ui/shared/internal/SharedModule.java

```diff
@@ -203,6 +204,7 @@ protected void configure() {
 				expose(ReferenceSearchViewPage.class);
 			}
 		});
+		bind(WorkspaceLockAccess.class);
```

Comment:
i wonder if we have any scheme for ordering the stuff here

Reply:
I did not find an obvious spot, so I put it at the end.